tests/test_basic_twoword_classifier.py

```diff
@@ -14,15 +14,16 @@ class BasicTwoWordClassifierTest(unittest.TestCase):
     def setUp(self):
         self.word1 = torch.from_numpy(np.array([[1, 0, 0]], dtype='float32'))
         self.word2 = torch.from_numpy(np.array([[0, 1, 0]], dtype='float32'))
-        self.inputdim = 6
-        self.hiddendim = 6
+        self.input_dim = 6
+        self.hidden_dim = 6
         self.labels = 2
 
     def test_forward(self):
         """
         tests the classifier implemented in BasicTwoWordClassifier and the overridden method "forward"
-        prints the classifier to check whether the right number of layers exist and the right dimensions are included
+        checks whether the output layer is of the right size
         """
-        classifier = BasicTwoWordClassfier(input_dim=self.inputdim, hidden_dim=self.hiddendim, label_nr=self.labels)
-        classifier.forward(self.word1, self.word2)
-        print(classifier)
+        self.expected_size = torch.tensor(np.array([[0,1]])).shape
```

Comment:
class variable only make sense if they are defined in the setup method and will eventually be reused by another test or if they are very general. the result of a test will only be used in one specific test in most cases so I would not define it as a class variable here.